geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-api/src/main/scala/org/locationtech/geomesa/fs/storage/api/PathFilterFactory.scala

```diff
@@ -0,0 +1,16 @@
+/***********************************************************************
+ * Copyright (c) 2013-2024 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.fs.storage.api
+
+import org.apache.hadoop.fs.PathFilter
+import org.geotools.api.filter.Filter
+
+trait PathFilterFactory {
```

Comment:
maybe we can rename this to `PathFiltering` (or if you can think of a better name), so we don't have a FactoryFactory :P